client/components/content-elements/tce-video/edit/Toolbar.vue

```diff
@@ -31,9 +31,13 @@ export default {
     url: vm => get(vm.element, 'data.assets.url')
   },
   methods: {
-    save({ url, publicUrl }) {
+    save({ url }) {
       const element = cloneDeep(this.element);
-      set(element.data, 'assets.url', url);
+      if (url) set(element.data, 'assets.url', url);
+      else {
+        set(element.data, 'assets', null);
```

Comment:
This might break future functionality, i.e. adding thumbnail or similar.  Please override just url.